### ISSUE TITLE
Not Overriding

### ISSUE THREAD

**Issue by ElementalVenom** [Jessecar96/SteamBot#841]
Hey guys im kinda new to this whole thing. The only C# experice ive had is in XNA and Unity3D so you might have to dumb it down a bit. So i made a new .cs file in the same place that userhandler is located called VenomUserHandler. I then overrided some of the functions. But when I start the bot none of these functions work. For exsample, when i send a message it ALWAYS replys the default message. And when it boots up its not telling me "LOGGED IN" Like ive set it to. Any ideas?

Link to all the code inside VenomUserHandler.cd:
http://pastebin.com/DXVHX1jV

**Comment by Bottswana** [comment]
Have you changed the userhandler from the demo handler to your one in
settings.json?

FYI: questions like this should be asked on our subreddit, /r/SteamBot.

**Comment by ElementalVenom** [comment]
Im sorry i have absolutely no idea where to put stuff like this. Im not much into forums and such. Can you eleberate what you mean?

**Comment by ElementalVenom** [comment]
Nope it seems to still not be working.

**Comment by Bottswana** [comment]
I'm not sure which part you are questioning.

There is an argument in the settings.json file which defines which class is
used for your userhandler. For more information on this, see the FAQ
section.

For any issues that are not bugs in SteamBot (like this issue) we request
that you ask them on Reddit. This can be done at this link:
http://reddit.com/r/SteamBot

And selecting 'Submit a new text post' on the right.

**Comment by ElementalVenom** [comment]
Alright thanks. Sorry about that.

**Comment by BlueRaja** [comment]
#427
